blogs/cli/index.md

```diff
@@ -0,0 +1,444 @@
+
+Behind the scenes in our scenario, we used to have a deny all on the pod label:
+```
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: deny-nodejs
+  namespace: connectivity-scenario
+spec:
+  podSelector:
+    matchLabels:
+      app: nodejs-basic
+  policyTypes:
+    - Ingress
+    - Egress
+```
+
+Once you updated your policies, you can give another try to your route until you fix the issue:
+![cli ok](./images/connectivity_scenario_cli_ok.png)
+
+The drop cause will dissapear and your route should open correctly now. On top of that, you can ensure that the Round Trip Time is correct. 
+If you are still experienting issues with the route, you may update / get rid of the filter(s) and play with live filtering.
+
+- While running a capture, you can place additionnal live filters to the view by simply typing keywords on your keyboard such as `nodejs`:
+![cli live filter](./images/connectivity_scenario_cli_live_filter.png)
+
+- And add as many keywords as you want by pressing enter:
+![cli live filters](./images/connectivity_scenario_cli_live_filters.png)
+
+- If your deployment involve some DNS resolution, you can switch to the DNS display and show latencies and errors:
+Once you updated your policies, you can give another try to your route until you fix the issue:
+![cli dns](./images/connectivity_scenario_cli_dns.png)
+
+  Those `NxDomain` errors could be from a misconfiguration in your app deployment trying to reach the wrong domain.
+
+Once you are done, simply press `CTRL + C` to exit. Your capture will be copied to your local machine for post mortem analysis.
+
+That's the end of the first scenario about connectivity checks !
+
+#### Identify unknown traffic content
+
+In this next scenario, we are going to dig into unknown packets using unencrypted traffic on `TCP` protocol on port `80`. We usually avoid such usage on production cluster and would like to understand who's involved in these.
+
+For the demo purepose, we are using again a nodejs basic deployment, exposing unencrypted traffic on port 80 in `packet-capture-scenario` namespace.
+
+Let's start a packet capture using the following command:
+```
+oc netobserv packets tcp,80
+```
+
+Similarly to the previous scenario, the script will connect to your cluster and start deploying eBPF agents and collector pod but capturing full packet content this time.
+
+Once first packet is captured, you will see a table with the amount of packets and bytes:
+
+![cli packets](./images/packet_capture_scenario_cli.png)
+
+Let the capture run enough time to get some packets and press `CTRL + C` to exit.
+Open the generated pcap file using Wireshark and look for your IP as source for example:
+
+![wireshark unencrypted packets](./images/wireshark_unencrypted_pcap.png)
+
+This time, we can see the HTTP content and see the request URL and response. Since we have the entire payload, we cand check for any authentification header, cookie, encoding and so on that may be wrong. 
```

Comment:
```suggestion
This time, we can see the HTTP content and see the request URL and response. Since we have the entire payload, we can check for any authentication header, cookie, encoding and so on that may be wrong. 
```